arches/app/models/migrations/9477_fix_for_spatial_view_dbf_function_table_alias_error.py

```diff
@@ -0,0 +1,258 @@
+                                loop
+                                    tmp_nodegroupid_slug := __arches_slugify(n.nodegroupid::text);
+                                    node_create = node_create ||
+                                        format(' ,__arches_agg_get_node_display_value(distinct "tile_%s".tiledata, %L::uuid) as %s '
+                                            ,tmp_nodegroupid_slug
+                                            ,n.nodeid::text
+                                            ,__arches_slugify(n.name)
+                                        );
+
+                                    if tile_create not like (format('%%tile_%s%%',tmp_nodegroupid_slug)) then
+                                        tile_create = tile_create ||
+                                            format(' left outer join tiles "tile_%s" on r.resourceinstanceid = "tile_%s".resourceinstanceid
+                                                and "tile_%s".nodegroupid = ''%s''::uuid ',
+                                            tmp_nodegroupid_slug,
+                                            tmp_nodegroupid_slug,
+                                            tmp_nodegroupid_slug,
+                                            n.nodegroupid::text);
+                                    end if;
+
+                                end loop;
+
+                                insert_attr := format(
+                                    '
+                                    insert into %s
+                                        select
+                                            r.resourceinstanceid
+                                            %s
+                                        from resource_instances r
+                                            join geojson_geometries geo on geo.resourceinstanceid = r.resourceinstanceid
+                                                and geo.nodeid = %L
+                                            %s
+                                        group by
+                                            r.resourceinstanceid
+                                        having r.resourceinstanceid = %L::uuid
+                                    ',
+                                    att_table_name,
+                                    node_create,
+                                    spv.geometrynodeid::text,
+                                    tile_create,
+                                    trigger_tile.resourceinstanceid::text);
+
+                            end;
+                            execute insert_attr;
+
+                        end;
+                    end loop;
+
+                    return trigger_tile;
+                end;
+
+            $BODY$;
+        """
+
+    reverse_sql_string = """
+            DROP FUNCTION IF EXISTS public.__arches_create_spatial_view_attribute_table;
+            DROP FUNCTION IF EXISTS public.__arches_trg_fnc_update_spatial_attributes;

```

Comment:
Could you also return these functions to their previous state rather than dropping them? It looks like they are also defined in the 8042_3_spatialview_db_functions migration.

Reply:
Function now returning to previous state, rather than it getting dropped.